tests/integration/test_chat_async.py

```diff
@@ -295,6 +312,12 @@ async def test_chat_message_proliferates_to_users_who_connected_before_room_was_
     # User sends message
     await communicator.send_json_to({'message_type': 'chat', 'room': room.pk, 'text': 'hello'})
 
+    # Wait for message object to be created...
+    await asyncio.sleep(0.1)
+
+    # Message is marked as read by the sender
+    await _get_message_object(user=user, users_who_viewed__in=[user])
+
     # Other user receives message
     response = await other_communicator.receive_json_from()
```

Comment:
Would be great to make sure that guy is coming back through the websocket as well, since that's what will immediately update the front end.